src/components/pages/packages/flyouts/newPackage/newPackage.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import {  LinkedComponent} from 'utilities';
+
+import './newPackage.css';
+
+export class NewPackage extends LinkedComponent {
+// TODO
+
+render () {
+  return ([]);
+}
```

Comment:
nit: spacing... but yes, I know this whole file will change in your next checkin.

Reply:
fixed